airbyte-webapp/src/pages/DestinationPage/pages/CreateDestinationPage/components/DestinationForm.tsx

```diff
@@ -58,9 +58,8 @@ const DestinationForm: React.FC<IProps> = ({
       afterSelectConnector();
     }
 
-    analyticsService.track("New Destination - Action", {
-      action: "Select a connector",
-      connector_destination_definition: connector?.name,
+    trackNewDestinationAction("Select a connector", {
+      connector_destination: connector?.name,
```

Comment:
The PM's spec for these events calls this event `connector_destination`. Not all destination events are using this (yet) but at least it makes these actions consistent.